ebcl/common/fake.py

```diff
@@ -145,22 +145,22 @@ def _special_folders(self, chroot: str, mount: bool) -> None:
             self.run_sudo(cmd, cwd=chroot, check=False)
 
         files = [
-            ('etc/resolv.conf', 'etc/resolv.conf'),
-            ('etc/gai.conf', 'etc/gai.conf'),
-            ('proc/mounts', 'etc/mtab'),
+            ('etc/resolv.conf', 'etc/resolv.conf', 'etc/btb.resolv.conf'),
+            ('etc/gai.conf', 'etc/gai.conf', 'etc/btb.gai.conf'),
+            ('proc/mounts', 'etc/mtab', 'etc/btb.mtab'),
         ]
 
-        for (src, tgt) in files:
+        for (src, tgt, backup) in files:
             target = os.path.join(chroot, tgt)
+            path = str(Path(target).parent)
 
             if mount:
-                path = str(Path(target).parent)
                 self.run_sudo(f'mkdir -p {path}', check=False)
-                cmd = f'cp /{src} {target}'
+                self.run_sudo(f'mv {path} {backup}', check=False)
+                self.run_sudo(f'cp /{src} {target}', cwd=chroot, check=False)
```

Comment:
Can we please add the slash to the definition of files above? That increases readbilty a lo